model/src/main/proto/arguments.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// Represents the arguments associated with ExitProfileDialogFragment.
+message ExitProfileDialogArguments {
+  // The last checked menu item needs to be restored or not.
```

Comment:
This documentation comment overall may benefit from more context: what does 'restore' mean & what menu is being described here?

Reply:
Provided more context on what does restore actually mean.